codeserver/c9s-python-3.9/nginx/serverconf/proxy.conf.template

```diff
@@ -1,13 +1,13 @@
 ###############
-# api calls from probes get to VSCode /healthz endpoint
+# api calls from probes get to Code-Server /healthz endpoint
```

Comment:
```suggestion
# api calls from probes get to Code Server /healthz endpoint
```

Code Server docs seem to either say "Code Server", capitalized, with space, or "code-server", lowercase and with hyphen (same as the cli command is named). I think we should not create our own hybrid.